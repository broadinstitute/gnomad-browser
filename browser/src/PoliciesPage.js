import React from 'react'
import styled from 'styled-components'

import { PageHeading } from '@gnomad/ui'

import policiesContent from '../about/policies.md'

import DocumentTitle from './DocumentTitle'
import InfoPage from './InfoPage'
import MarkdownContent from './MarkdownContent'

const PoliciesPage = styled(InfoPage)`
  h2:not(:first-child) {
    padding-top: 1.5rem;
  }
`

export default () => (
  <PoliciesPage>
    <DocumentTitle title="Policies" />
    <PageHeading>Policies</PageHeading>

    <MarkdownContent dangerouslySetInnerHTML={{ __html: policiesContent.html }} />
  </PoliciesPage>
)
