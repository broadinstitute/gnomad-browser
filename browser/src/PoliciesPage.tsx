import React from 'react'
import styled from 'styled-components'

import { PageHeading } from '@gnomad/ui'

import termsContent from '../about/policies/terms.md'
import privacyPolicy from '../about/policies/policies_gnomAD_privacy_DRAFT.pdf'
import policiesContent from '../about/policies/policies.md'

import DocumentTitle from './DocumentTitle'
import InfoPage from './InfoPage'
import MarkdownContent from './MarkdownContent'

const PoliciesPage = styled(InfoPage)`
  h2 {
    font-size: 1.5em;
    font-weight: bold;
  }

  h2:not(:first-child) {
    padding-top: 1.5rem;
  }
`
// TODO: formatting from 'MarkdownContent' breaks a-tags that link to a PDF loaded with
//   webpack, which is how the current Privacy Policy is being loaded.
//   Workaround for now to get this up for GCBR application, will review later.
const PrivacyPolicyWrapper = styled.div`
  p {
    margin-top: 15px;
    margin-bottom: 15px;
    line-height: 1.4;
  }

  a {
    color: #428bca;
    text-decoration: none;
  }
`

export default () => (
  <PoliciesPage>
    <DocumentTitle title="Policies" />
    <PageHeading>Policies</PageHeading>

    <MarkdownContent dangerouslySetInnerHTML={{ __html: termsContent.html }} />

    <br />

    <PrivacyPolicyWrapper>
      <h2>gnomAD Privacy Policy (Draft)</h2>
      <p>
        gnomADâ€™s draft Privacy Policy that outlines what data we store while you are using our
        website and when you communicate with us, can be found{' '}
        <a href={privacyPolicy} target="_blank" rel="noreferrer">
          here
        </a>
        . It is currently undergoing legal review and is not yet final.
      </p>
    </PrivacyPolicyWrapper>

    <br />

    <MarkdownContent dangerouslySetInnerHTML={{ __html: policiesContent.html }} />
  </PoliciesPage>
)
