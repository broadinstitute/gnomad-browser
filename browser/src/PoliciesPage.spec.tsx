import { describe, expect, test } from '@jest/globals'
import 'jest-styled-components'

import React from 'react'
import renderer from 'react-test-renderer'

import PoliciesPage from './PoliciesPage'

import { withDummyRouter } from '../../tests/__helpers__/router'

test('Policies Page has no unexpected changes', () => {
  const tree = renderer.create(withDummyRouter(<PoliciesPage />))
  expect(tree).toMatchSnapshot()
})
