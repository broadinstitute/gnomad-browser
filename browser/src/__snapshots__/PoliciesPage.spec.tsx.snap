// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Policies Page has no unexpected changes 1`] = `
.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  padding-bottom: 0.5em;
  border-bottom: 1px solid #ccc;
  margin: 0.67em 0;
  font-size: 14px;
}

.c2 {
  margin: 0;
}

.c3 {
  font-size: 16px;
}

.c3 h1,
.c3 h2,
.c3 h3 {
  font-weight: bold;
}

.c3 h1 {
  font-size: 2em;
}

.c3 h2 {
  font-size: 1.5em;
}

.c3 p {
  margin-top: 15px;
  margin-bottom: 15px;
  line-height: 1.4;
}

.c3 a {
  color: #428bca;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c3 img {
  max-width: 100%;
}

.c3 blockquote {
  margin: 0 0 0 10px;
  font-size: 14px;
  font-style: italic;
  line-height: 1.4;
}

.c3 ol,
.c3 ul {
  padding-left: 20px;
  margin: 1em 0;
}

.c3 li {
  margin-bottom: 0.5em;
}

.c3 table {
  border-collapse: collapse;
  border-spacing: 0;
}

.c3 td {
  padding: 0.5em 10px 0.5em 0;
  border-bottom: 1px solid #ccc;
  font-weight: normal;
  text-align: left;
}

.c3 th {
  padding: 0.5em 10px 0.5em 0;
  border-bottom: 1px solid #000;
  background-position: center right;
  background-repeat: no-repeat;
  font-weight: bold;
}

.c0 {
  box-sizing: border-box;
  width: 100%;
  max-width: 1200px;
  padding: 0 15px;
  margin: 0 auto 40px;
  font-size: 14px;
  font-size: 16px;
}

.c0 p {
  margin-bottom: 1em;
  line-height: 1.4;
}

.c0 h2 {
  font-size: 1.5em;
  font-weight: bold;
}

.c0 h2:not(:first-child) {
  padding-top: 1.5rem;
}

.c4 p {
  margin-top: 15px;
  margin-bottom: 15px;
  line-height: 1.4;
}

.c4 a {
  color: #428bca;
  -webkit-text-decoration: none;
  text-decoration: none;
}

@media (max-width:900px) {
  .c1 {
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
    -webkit-align-items: center;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
    border-bottom: none;
  }
}

@media (max-width:900px) {
  .c2 {
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
    -webkit-align-items: center;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
    padding-bottom: 0.5em;
    border-bottom: 1px solid #ccc;
    margin-bottom: 0.5em;
  }
}

<div
  className="c0"
>
  <div
    className="c1"
  >
    <h1
      className="c2"
    >
      Policies
    </h1>
  </div>
  <div
    className="c3"
    dangerouslySetInnerHTML={
      {
        "__html": "## Terms of use

All data here are released openly and publicly for the benefit of the wider biomedical community. You can freely download and search the data, and we encourage the use and publication of results generated from these data. **There are absolutely no restrictions or embargoes on the publication of results derived from gnomAD data**. However, we encourage you to [contact the consortium](mailto:gnomad@broadinstitute.org) before embarking on large-scale analyses to check if your proposed analysis overlaps with work currently underway by the gnomAD consortium. All users of gnomAD data agree to not attempt to reidentify participants.

This data set has been subjected to extensive quality control, but variant calling and filtering from short-read sequencing data is an imperfect and probabilistic process, so many errors no doubt remain. If you spot any results that seem impossible, or suggest some kind of serious processing or variant-calling artifact, don't panic: use the "report variant" form on the corresponding variant page or [email us](mailto:gnomad@broadinstitute.org) to let us know, and we'll do our best to address it.

The primary data from the gnomAD exomes and genomes are available free of restrictions under the [Creative Commons Zero Public Domain Dedication](https://creativecommons.org/publicdomain/zero/1.0/). This means that you can use it for any purpose without legally having to give attribution. However, we request that you actively acknowledge and give attribution to the gnomAD project, and link back to the relevant page, wherever possible. Attribution supports future efforts to release other data. It also reduces the amount of "orphaned data", helping retain links to authoritative sources.

Screenshots of the website may also be used without restriction. As with any use of gnomAD data, we request that you actively acknowledge and give attribution to the gnomAD project, and link back to the relevant page, wherever possible.

Some annotations may have restrictions on usage. For instance, SpliceAI annotations have been computed by Illumina and are provided with permission under a CC BY NC 4.0 license for academic and non-commercial use [SpliceAI](https://github.com/Illumina/SpliceAI). It is the responsibility of users to abide by all relevant licensing requirements.

## Citation in publications

We request that any use of data obtained from the gnomAD browser cite [the gnomAD flagship paper](https://broad.io/gnomad_lof) and any online resources that include the data set provide a link to the browser. For use of the SV data, we request cite [this paper](https://broad.io/gnomad_sv) for use of the SV data.

There is no need to include us as authors on your manuscript, unless we contributed specific advice or analysis for your work.
",
      }
    }
    onClick={[Function]}
  />
  <br />
  <div
    className="c4"
  >
    <h2>
      gnomAD Privacy Policy (Draft)
    </h2>
    <p>
      gnomAD’s draft Privacy Policy that outlines what data we store while you are using our website and when you communicate with us, can be found
       
      <a
        href="policies_gnomAD_privacy_DRAFT.pdf"
        rel="noreferrer"
        target="_blank"
      >
        here
      </a>
      . It is currently undergoing legal review and is not yet final.
    </p>
  </div>
  <br />
  <div
    className="c3"
    dangerouslySetInnerHTML={
      {
        "__html": "## gnomAD Ethics Policy

Data included in the gnomAD database has met our ethical requirements including the Principal Investigator (PI) of all shared datasets signing our [memorandum of understanding (MoU)](https://gnomad.broadinstitute.org/mou), and careful review of each study’s consent by Broad-employed regulatory experts to ensure data is appropriately consented for sharing.

## gnomAD Data Preservation Policy

While we hope gnomAD exists for decades to come, we recognize the importance of having a plan in place for preserving gnomAD datasets. Through our [collaboration](https://gnomad.broadinstitute.org/news/2020-10-open-access-to-gnomad-data-on-multiple-cloud-providers/) with Google Cloud Platform (GCP), Microsoft’s Azure, and Amazon Web Services (AWS), we have ensured that the public would continue to have access to any past gnomAD datasets as long as at least one of those public dataset hosting entities remains in business. Additionally we will do everything possible to sustain the existing browser for as long as financial/staffing commitments allow.

## gnomAD Open Science Policy

The gnomAD team has a firm commitment to open science. This includes, but is not limited to, making our data and code open-source, posting pre-prints, and prioritizing publishing in journals that support open access.

We request that developers integrating gnomAD data in their tools include a statement acknowledging the inclusion of gnomAD data (e.g., "This tool is powered by the gnomAD v4.1 release data."). However, to avoid confusion and misattribution, we ask that you refrain from incorporating "gnomAD"/"Genome Aggregation Database" into the name of your tool and from using the gnomAD logo without permission.

## Data Generation

A full description of the methods used to aggregate and call variants across the exomes and genomes in this project will be provided shortly. In brief: we pulled raw data together from as many exomes and genomes as we could get our hands on, aligned and processed each of these data types through unified processing pipelines based on Picard, and performed variant calling with the GATK HaplotypeCaller following GATK best practices. Processing and variant calling at this enormous scale was only possible thanks to the hard work of the Broad Institute's Data Sciences Platform, and the Intel GenomicsDB team. Downstream analysis relied heavily on the [Hail](https://hail.is/) toolkit.
",
      }
    }
    onClick={[Function]}
  />
</div>
`;
